modules/statistics/AnalyticsAdapter.js

```diff
@@ -29,12 +29,12 @@ class CacheAnalytics extends AnalyticsAbstract {
 
     /**
      * Cache analytics event.
-     * @param {String} action the name of the event
+     * @param {String} eventName the name of the event
      * @param {Object} data can be any JSON object
      */
-    sendEvent(action, data = {}) {
+    sendEvent(eventName, data = {}) {
         this.eventCache.push({
-            action,
+            eventName,
```

Comment:
This change makes so much sense, especially for consistency, I have to point out how much sense it makes.